frontend/src/pages/document.tsx

```diff
@@ -17,15 +17,67 @@ import { WebvttExportModal } from '../editor/webvtt_export';
 import { showModal } from '../components/modal';
 import { Tooltip } from '../components/tooltip';
 import { SpeakerColorsProvider } from '../editor/speaker_colors';
+import { Input } from '../components/form';
+import { BiPencil } from 'react-icons/bi';
 
 const LazyDebugPanel = lazy(() =>
   import('../editor/debug_panel').then((module) => ({ default: module.DebugPanel })),
 );
 
+function DocumentTitle({ name, onChange }: { name: string; onChange: (newTitle: string) => void }) {
+  const [editable, setEditable] = useState(false);
+
+  if (editable) {
+    return (
+      <form
```

Comment:
This should prevent submitting an empty name

Reply:
Do we want even more normalization?
Probably only whitespace is also not really something we want to allow. 
Do we want to allow newlines?